Demo/Common/IntegrationExamples/DropIn.swift

```diff
@@ -0,0 +1,114 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import AdyenActions
+import AdyenCard
+import AdyenComponents
+import AdyenDropIn
+import AdyenNetworking
+import AdyenSession
+import UIKit
+
+internal final class DropIn: DropInExampleProtocol {
```

Comment:
Let's name these `example`s too such as `DropInExample`. Dropin could be too broad/confusing.

Reply:
Nice point :). Made the change here: da093c2e272a314effba7b3c43882271964a0529